cosmic-usage-ui/src/main/resources/public/app/main.js

```diff
@@ -27,6 +27,19 @@ const app = {
     generateReportButton: '#ui-generate-report-btn',
     printingHeadersContainer: '#ui-printing-headers',
     domainsTable: '#ui-domains-table',
+    domainsTableDomainHeadId: '#ui-domains-table-domain-head',
+    domainsTableCpuHeadId: '#ui-domains-table-cpu-head',
+    domainsTableMemoryHeadId: '#ui-domains-table-memory-head',
+    domainsTableStorageHeadId: '#ui-domains-table-storage-head',
+    domainsTablePublicIpHeadId: '#ui-domains-table-public-ip-head',
+
+    // Attributes
+    domainTableHeaderState: 'data-state',
```

Comment:
Combined with `jQuery.data(...)` we can eliminate the `data-` part of the string.